packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -425,21 +425,33 @@ class _AuthenticatorState extends State<Authenticator> {
     if (cognitoPlugin == null) {
       return const ['auth.plugins.Auth.Default'];
     }
+    if (cognitoPlugin.usernameAttributes == null) {
+      missingValues.add('\n - usernameAttributes');
+    }
     if (cognitoPlugin.signupAttributes == null) {
-      missingValues.add('auth.plugins.Auth.Default.signUpAttributes');
+      missingValues.add('\n - signUpAttributes');
```

Comment:
The formatting updates here (and some below) are to make this easier to read. See before/after below.

### Before

![Screen Shot 2021-12-15 at 5 10 11 PM](https://user-images.githubusercontent.com/20613561/146272841-a091694c-1125-4cad-b8b9-7e1dcc902c64.png)
<img width="674" alt="Screen Shot 2021-12-15 at 5 10 24 PM" src="https://user-images.githubusercontent.com/20613561/146272843-6c7a328f-0380-40aa-81f0-927c1b4bfb1c.png">

### After

![Screen Shot 2021-12-15 at 5 03 50 PM](https://user-images.githubusercontent.com/20613561/146272725-7647c1a1-152b-4442-9e36-b78d100bdaaf.png)
<img width="678" alt="Screen Shot 2021-12-15 at 5 03 34 PM" src="https://user-images.githubusercontent.com/20613561/146272723-8d15631b-6dfc-48cf-890d-88c09f68ed82.png">